server/src/main/scala/services/graphstages/ArrivalsGraphStage.scala

```diff
@@ -284,6 +288,11 @@ class ArrivalsGraphStage(name: String = "",
       pushIfAvailable(toPush, outArrivalsDiff)
     }
 
+    def updateFilteredIncomingWithTotalPaxSource(filteredIncoming: SortedMap[UniqueArrival, Arrival], feedSource: FeedSource): SortedMap[UniqueArrival, Arrival] =
```

Comment:
Great that we're setting the TotalPax from the appropriate source here in one place.
I think we need to stop removing the transit passengers though. In fact I've just realised it would make sense to add a transit pax field to the TotalPax class, so that we can track where different transit pax numbers came from too.

Reply:
Yeah , I was thinking move actual pax and transit pax in totalPax source as separate ticket as this is already huge change to merge.